packages/cli/src/controllers/project.controller.ts

```diff
@@ -98,10 +100,7 @@ export class ProjectController {
 		for (const pr of relations) {
 			const result: ProjectRequest.GetMyProjectsResponse[number] = Object.assign(
 				this.projectRepository.create(pr.project),
-				{
-					role: pr.role,
-					scopes: req.query.includeScopes ? ([] as Scope[]) : undefined,
-				},
+				{ role: pr.role, scopes: [] },
```

Comment:
Was the `includeScopes` not used in the FE?

Reply:
nop. I checked the code, and checked with Danny. FE was always passing in `includeScopes` as `true`.